components/dashboard/src/Menu.tsx

```diff
@@ -185,7 +188,7 @@ export default function Menu() {
                     <Link to="/">
                         <img src={gitpodIcon} className="h-6" />
                     </Link>
-                    <div className="ml-2 text-base">
+                    {!isMinimalUI && <div className="ml-2 text-base">
```

Comment:
**issue**: There's a small shift when removing the team element. Could be related to [the padding above](https://github.com/gitpod-io/gitpod/pull/4571/files#diff-64248da95f399a1cfe99b03aa662caf313faf9b7016999111605da5d61f1971dR131)? 🤷 

| Team (New) | Team (Existing) |
|-|-|
| <img width="399" alt="team-new" src="https://user-images.githubusercontent.com/120486/123415452-f3c15100-d5bd-11eb-8daa-979e9c5a3e4e.png"> | <img width="399" alt="team-existing" src="https://user-images.githubusercontent.com/120486/123415460-f4f27e00-d5bd-11eb-99cf-2e23cea4da66.png"> |

Reply:
Hahaha, nice catch! (~~I was hoping nobody would notice so that I could be lazy about it 😅~~)